src/pages/webhooks/commands.md

```diff
@@ -7,20 +7,41 @@ keywords:
 
 # Webhooks commands
 
+Adobe Commerce provides the following commands to configure and process webhooks:
+
+* [webhooks:info](#display-the-payload-of-a-webhook)
+* [webhooks:list:all](#return-a-list-of-supported-webhook-event-names)
+* [webhooks:list](#return-a-list-of-subscribed-webhooks)
+* [webhooks:generate:module](#generate-plugins)
+
 ## Display the payload of a webhook
 
 The `webhooks:info` command returns the payload of the specified webhook. You can optionally specify the depth of the payload to reduce the amount data returned.
-The actual payload can be different from that shown in the `webhooks:info` as it generates in runtime based on the current state of objects and variables.
+
+<InlineAlert variant="info" slots="text" />
+
+The actual payload of a transmitted webhook can be different from that returned by the `webhooks:info`. The command response is based on the current state of objects and variables.
+
+The value of the `webhook-name` argument must be a valid Commerce event name. The value must use the following pattern:
+
+```text
+<type>.<event_name>
+```
+
+where:
+
+*  `type` specifies the origin of the event. Specify `observer` if the event is emitted by a Commerce observer, or specify `plugin` if the event is emitted by a plugin.
```

Comment:
`or specify `plugin` if the event is emitted by a plugin.`

I would rephrase it to 
`or specify `plugin` if the event is based on methods of Resource Models or Api Interfaces`

As it's not emitted by the plugin, we are generating plugins for Resource Models or API Interfaces based on those event names for.